packaging/docker/samples/python/docker-compose.yml

```diff
@@ -19,18 +19,34 @@
 
 version: '3'
 services:
-  fdb:
-    image: foundationdb:5.2.5
+  # Specify three fdbserver processes.
+  fdb-coordinator:
+    image: foundationdb:6.1.8
     environment:
       FDB_COORDINATOR: fdb-coordinator
-  fdb-coordinator:
-    image: foundationdb:5.2.5
+  fdb-server-1:
+    depends_on:
+      - fdb-coordinator
+    image: foundationdb:6.1.8
+    environment:
+      FDB_COORDINATOR: fdb-coordinator
+  fdb-server-2:
+    depends_on:
+      - fdb-coordinator
+    image: foundationdb:6.1.8
     environment:
       FDB_COORDINATOR: fdb-coordinator
+
+  # Bring up the application so that it depends on the cluster.
   app:
+    depends_on:
+      - fdb-coordinator
+      - fdb-server-1
+      - fdb-server-2
     build:
       context: app
     ports:
       - 5000:5000
     environment:
-      FDB_COORDINATOR: fdb-coordinator
\ No newline at end of file
+      FDB_COORDINATOR: fdb-coordinator
+      FDB_COORDINATOR_PORT: 4550
```

Comment:
Ah, good catch. I reverted this to not include the port API environment variable, and then this exposed one other issue in `create_cluster_file.bash` if the `FDB_COORDINATOR_PORT` environment variable isn't set. When running in the image created in this repo, that should never happen, but it can happen if someone has copied over the script to a different image (as in this example), so that script now also will supply a default value of 4500. I've re-run that sample and it works now.